src/views/AddItem.jsx

```diff
@@ -1,3 +1,81 @@
+import { useState } from 'react';
+import { addItem } from '../api';
+import { useStateWithStorage } from '../utils';
+
+import './AddItem.css';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [itemToAdd, setItemToAdd] = useState({});
+	const [listToken, setListToken] = useStateWithStorage(
+		'my test list',
+		'tcl-shopping-list-token',
+	);
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const docRef = await addItem(listToken, {
+				itemName: itemToAdd.itemName,
+				daysUntilNextPurchase: itemToAdd.buyingFrequency,
+			});
+
+			docRef && alert(`${itemToAdd.itemName} was saved to the database`);
+		} catch (err) {
+			console.log(err);
+			err && alert('item not saved, pls try again');
+		}
+	};
+
+	const handleChange = (e) => {
+		const { name, value } = e.target;
+		setItemToAdd({ ...itemToAdd, [name]: value });
+	};
+
+	return (
+		<>
+			<form className="add-item-form" method="post" onSubmit={handleSubmit}>
+				<label htmlFor="item-name">Item name:</label>
+				<input
+					type="text"
+					name="itemName"
+					id="item-name"
+					onChange={handleChange}
+				/>
+				<span>How soon will you buy this again</span>
+				<section>
+					<div>
+						<input
+							type="radio"
+							value={7}
+							name="buyingFrequency"
+							id="soon"
+							onChange={handleChange}
+						/>
```

Comment:
We need to select the first radio button by default, see wireframes here: https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/add-item.png